Cargo.toml

```diff
@@ -34,7 +34,7 @@ wasmparser = "=0.116.1"
 
 # NB: When updating, also update the version in rs-soroban-env dev-dependencies
 [workspace.dependencies.stellar-xdr]
-version = "=22.0.0-rc.1"
+version = "=22.0.0-rc.1.1"
```

Comment:
This version just adds a cli command, so we don't need to re-release env right?

Reply:
We need to re-release the env and sdk, for the cli to import this because of how the env and sdk pin the xdr version.